x-pack/legacy/plugins/siem/public/components/charts/barchart.tsx

```diff
@@ -131,8 +131,4 @@ export const BarChartComponent = ({
   );
 };
 
-BarChartComponent.displayName = 'BarChartComponent';
```

Comment:
do the recent tooling changes make explicitly setting `displayName` unnecessary in cases like this?

Reply:
Yes with @patrykkopycinski npm library

Reply:
the library wasn't added @XavierM :( but the good news is that if we declare component as ```const component:React.FC<>``` and then declare ```const MemoComponent = React.memo(Component)``` it will get the `displayName` automatically